lib/common.js

```diff
@@ -53,7 +53,13 @@ export default [
       "generator-star-spacing": ["error", { before: false, after: true }],
       "arrow-spacing": ["error", { before: true, after: true }],
       "keyword-spacing": ["error", { before: true, after: true }],
-      "indent": [
+      "no-unused-vars": [
```

Comment:
added this cause its asking for variables in catches

Reply:
I'd prefer turning on these with the `^_` regex instead:
- https://eslint.org/docs/latest/rules/no-unused-vars#destructuredarrayignorepattern
- https://eslint.org/docs/latest/rules/no-unused-vars#argsignorepattern
- https://eslint.org/docs/latest/rules/no-unused-vars#caughterrorsignorepattern
- https://eslint.org/docs/latest/rules/no-unused-vars#reportusedignorepattern